spec/views/posts/index.html.erb_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Index Page Features', type: :feature do
+  before(:each) do
+    visit users_path
+
+    @photo = 'https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/160/microsoft/58/ballot-box-with-x_2612.png'
+
+    @user1 = User.create!(name: 'John Doe', bio: 'This is John Doe bio', photo: @photo, postsCounter: 16)
+    @user2 = User.create!(
+      name: 'Michal Smith', bio: 'This is Michal Smith bio', photo: @photo, postsCounter: 3
+    )
+
+    @post1 = @user1.posts.create!(
+      title: 'Education', text: 'This is first Educat post!', commentsCounter: 0, likesCounter: 0
+    )
+    @post2 = @user1.posts.create!(
+      title: 'Sport', text: 'This is second Sport post!', commentsCounter: 0, likesCounter: 0
+    )
+    @post3 = @user2.posts.create!(
+      title: 'Business', text: 'This is third!', commentsCounter: 0, likesCounter: 0
+    )
+
+    # Comments
+
+    @user1.comments.create!(text: 'Really nice post', post: @post1)
+    @user2.comments.create!(text: 'Amezing', post: @post1)
+    @user2.comments.create!(text: 'Fantastic', post: @post2)
+
+    # likes
+
+    @user1.likes.create!(post: @post1)
+    @user1.likes.create!(post: @post2)
+    @user2.likes.create!(post: @post1)
+    @user2.likes.create!(post: @post3)
+
+    visit user_posts_path(@user1)
+  end
+
+  describe 'I can see' do
+    it 'the user\'s profile picture' do
+      image = page.find('img')['src']
+      expect(image).to eql @photo
+    end
+
+    it 'the user\'s username' do
+      expect(page).to have_content 'John Doe'
+    end
+
+    it 'the number of posts the user has written' do
+      expect(page).to have_content 'Number of posts: 18'
+    end
+
+    it 'some post\'s title.' do
+      expect(page).to have_content 'Education'
+      expect(page).to have_content 'Sport'
+    end
+
+    it 'some of the post\'s body.' do
+      expect(page).to have_content 'This is first Educat post!'
+      expect(page).to have_content 'This is second Sport post!'
+    end
+
```

Comment:
- [ ] nice work. please kindly add Tests that asserts you can see the first comments on a post